central/imagecomponent/v2/datastore/datastore_sac_test.go

```
@@ -0,0 +1,436 @@
+//go:build sql_integration
+
+package datastore
+
+import (
+	"context"
+	"testing"
+
+	graphDBTestUtils "github.com/stackrox/rox/central/graphdb/testutils"
+	"github.com/stackrox/rox/generated/storage"
+	"github.com/stackrox/rox/pkg/features"
+	"github.com/stackrox/rox/pkg/fixtures"
+	"github.com/stackrox/rox/pkg/sac"
+	"github.com/stackrox/rox/pkg/sac/resources"
+	sacTestUtils "github.com/stackrox/rox/pkg/sac/testutils"
+	"github.com/stackrox/rox/pkg/scancomponent"
+	"github.com/stretchr/testify/suite"
+)
+
+func TestImageComponentV2DataStoreSAC(t *testing.T) {
+	suite.Run(t, new(componentV2DataStoreSACTestSuite))
+}
+
+type componentV2DataStoreSACTestSuite struct {
+	suite.Suite
+
+	testGraphDatastore  graphDBTestUtils.TestGraphDataStore
+	imageComponentStore DataStore
+
+	imageTestContexts map[string]context.Context
+}
+
+func (s *componentV2DataStoreSACTestSuite) SetupSuite() {
+	if !features.FlattenCVEData.Enabled() {
+		s.T().Setenv(features.FlattenCVEData.EnvVar(), "true")
+	}
+
+	var err error
+	s.testGraphDatastore, err = graphDBTestUtils.NewTestGraphDataStore(s.T())
+	s.Require().NoError(err)
+	pool := s.testGraphDatastore.GetPostgresPool()
+	s.imageComponentStore = GetTestPostgresDataStore(s.T(), pool)
+	s.imageTestContexts = sacTestUtils.GetNamespaceScopedTestContexts(context.Background(), s.T(), resources.Image)
+}
+
+func getImageComponentID(component *storage.EmbeddedImageScanComponent, imageID string) string {
+	componentID, _ := scancomponent.ComponentIDV2(component, imageID)
+	return componentID
```

why ignore err returned ?